vnext/Microsoft.ReactNative/Fabric/Composition/CompositionEventHandler.cpp

```diff
@@ -369,86 +395,104 @@ winrt::Windows::System::VirtualKeyModifiers GetKeyModifiers(uint64_t wParam) {
 int64_t CompositionEventHandler::SendMessage(HWND hwnd, uint32_t msg, uint64_t wParam, int64_t lParam) noexcept {
   switch (msg) {
     case WM_LBUTTONDOWN: {
-      auto pp = winrt::make<winrt::Microsoft::ReactNative::Composition::Input::implementation::PointerPoint>(
-          hwnd, msg, wParam, lParam, m_compRootView.ScaleFactor());
-      onPointerPressed(pp, GetKeyModifiers(wParam));
+      if (auto strongRootView = m_wkRootView.get()) {
```

Comment:
Did every single on of these need the strong pointer? Could have wrapped the whole function of it instead of each switch statement.

Reply:
There are potentially a lot of other window messages coming through here, so I wanted to avoid the strong pointer on every message.